packages/kubernetes-1.23/kubernetes-1.23.spec

```diff
@@ -89,17 +108,27 @@ export FORCE_HOST_GO=1
 make generated_files
 
 # Build kubelet with the target toolchain.
+%set_cross_go_flags
+unset CC
```

Comment:
`%set_cross_go_flags` brings the standard `GOFLAGS` and `GOLDFLAGS` into the environment. Afterwards, we need to unset the `CC` variable or else it'll fail to build the host codegen programs.

The `export %{kube_cc}` line below (unchanged) is what sets the cross-compiler in the way that the Makefile expects.